src/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -63,7 +63,7 @@ def _vgg16_data_prep(batch):
 
 def create(style_dataset, content_dataset, style_feature=None,
         content_feature=None, max_iterations=None, model='resnet-16',
-        verbose=True, batch_size = 6, **kwargs):
+        verbose=True, batch_size = 1, **kwargs):
```

Comment:
Why change the default `batch_size` to 1?

Reply:
When we tested style_transfer earlier this year, a batch_size of 1 produced a better result. Shreya and I discussed and decided that it was proper to change this here. (Plus, MPS defaults to a batch size of 1 currently so that makes things easier).